plugins/azure/virtualmachines/premiumSsdDisabled.js

```diff
@@ -0,0 +1,61 @@
+var async = require('async');
+
+var helpers = require('../../../helpers/azure/');
+
+module.exports = {
+    title: 'Premium SSD Disabled',
+    category: 'Virtual Machines',
+    description: 'Ensures that the Azure virtual machines are configured to use standard SSD disk volumes instead of premium SSD disk volumes.',
```

Comment:
```suggestion
    description: 'Ensures that the Azure virtual machines are configured to use standard SSD disk volumes instead of premium SSD disk volumes for managed disks.',
```